Makefile

```diff
@@ -39,6 +39,27 @@ fmt: ## Go fmt your code
 vet: ## Apply go vet to all go files
 	hack/go-vet.sh ./...
 
+.PHONY: test-e2e
+test-e2e: ## Run k8s specific e2e test
+	# KUBECONFIG and SSH_PK dirs needs to be mounted inside a container if tests are run in containers
+	go test -timeout 30m \
+		-v github.com/openshift/cluster-api-actuator-pkg/pkg/e2e \
+		-kubeconfig $${KUBECONFIG:-~/.kube/config} \
+		-machine-api-namespace openshift-machine-api \
+		-ginkgo.v \
+		-args -v 5 -logtostderr true
+
+.PHONY: k8s-e2e
+k8s-e2e: ## Run k8s specific e2e test
+	# KUBECONFIG and SSH_PK dirs needs to be mounted inside a container if tests are run in containers
+	go test -timeout 30m \
```

Comment:
I think the `test-e2e` and the `k8s-e2e` only differ in the namespace. We could create a Makefile macro here and reduce the duplication.

Reply:
Any example?

Reply:
We might have a single `make e2e-operator` or similar and run `make e2e-operator NAMESPACE=kube-system` so there is only a single point of entry. This will require to update the Prow job first. Let's do it in the next iteration so we can start adding new tests to the new framework.